core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -665,16 +663,9 @@ class KafkaConfig private(doLog: Boolean, val props: util.Map[_, _])
 
   def saslMechanismControllerProtocol: String = getString(KRaftConfigs.SASL_MECHANISM_CONTROLLER_PROTOCOL_CONFIG)
 
-  def controlPlaneListener: Option[EndPoint] = {
-    controlPlaneListenerName.map { listenerName =>
-      listeners.filter(endpoint => endpoint.listenerName.value() == listenerName.value()).head
-    }
-  }
-
   def dataPlaneListeners: Seq[EndPoint] = {
```

Comment:
Do we want to rename all the `dataPlane*` methods to remove the prefix? Perhaps that can be a separate JIRA ticket since it's not required for 4.0.

Reply:
file https://issues.apache.org/jira/browse/KAFKA-18362